pkg/kubectl/rollback.go

```diff
@@ -283,6 +287,83 @@ func (r *DaemonSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations ma
 	return rollbackSuccess, nil
 }
 
+type StatefulSetRollbacker struct {
+	c clientset.Interface
+}
+
+// toRevision is a non-negative integer, with 0 being reserved to indicate rolling back to previous configuration
+func (r *StatefulSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations map[string]string, toRevision int64, dryRun bool) (string, error) {
```

Comment:
Lots of code duplication with `(r *DaemonSetRollbacker) Rollback`. Would you de-duplicate it (or at least part of it)?